faker/providers/lorem/la/__init__.py

```diff
@@ -5,7 +5,17 @@
 
 
 class Provider(LoremProvider):
-    word_list = (
+
+    def __init__(self, ext_word_list=None):
+        #verify the type of the variable
```

Comment:
Wouldn't it make more sense to keep the existing `word_list` variable and then override it based on whether or not `ext_word_list` was provided?

```python
class Provider(LoremProvider):

    def __init__(self, ext_word_list=None):
        if ext_word_list is not None:
            self.word_list = ext_word_list

    word_list = (
        # existing list of words
```